collectors/plugins.d/pluginsd_parser.c

```diff
@@ -2474,6 +2474,25 @@ static inline PARSER_RC pluginsd_register_job(char **words __maybe_unused, size_
     return pluginsd_register_job_common(&words[2], num_words - 2, parser, words[1]);
 }
 
+static inline PARSER_RC pluginsd_dyncfg_reset(char **words __maybe_unused, size_t num_words __maybe_unused, PARSER *parser __maybe_unused) {
+    if (unlikely(num_words != (SERVING_PLUGINSD(parser) ? 1 : 2)))
+        return PLUGINSD_DISABLE_PLUGIN(parser, PLUGINSD_KEYWORD_DYNCFG_RESET, SERVING_PLUGINSD(parser) ? "expected 0 parameters" : "expected 1 parameter: plugin_name");
+
+    if (SERVING_PLUGINSD(parser)) {
+        unregister_plugin(parser->user.host->configurable_plugins, parser->user.cd->cfg_dict_item);
+        rrdpush_send_dyncfg_reset(parser->user.host, parser->user.cd->configuration->name);
+        parser->user.cd->configuration = NULL;
+    } else {
+        const DICTIONARY_ITEM *di = dictionary_get_and_acquire_item(parser->user.host->configurable_plugins, words[1]);
```

Comment:
Hi Timo. Not sure, but do you need later to do an `dictionary_acquired_item_release` ?

Reply:
Ah, sorry, `unregister_plugin` does it.